services/director-v2/src/simcore_service_director_v2/core/settings.py

```diff
@@ -239,6 +239,7 @@ class AppSettings(BaseApplicationSettings, MixinLoggingSettings):
         Field(
             ...,
             description="Base URL used to access the public api e.g. http://127.0.0.1:6000 for development or https://api.osparc.io",
+            deprecated=True,
```

Comment:
you can just remove that ENV.
This will need a MR in ops, but otherwise we will keep these forever.